src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -38,6 +38,10 @@ ucs_config_field_t uct_rc_mlx5_iface_config_table[] = {
    ucs_offsetof(uct_rc_mlx5_iface_config_t, super.mlx5_common),
    UCS_CONFIG_TYPE_TABLE(uct_ib_mlx5_iface_config_table)},
 
+  {"", "", NULL,
```

Comment:
how come there 2 configuration tables which can modify the same fields? super.super and super.mlx5_common are configure both here, and in lines 30, 38

Reply:
Each table has own fields. What do i miss?

Reply:
this new field accesses the whole 'super' field, while previous fields access parts of the super (eg super.mlx5_common)

Reply:
this new field accesses the whole 'super' field, while previous fields access parts of the super (eg super.mlx5_common)

Reply:
this new field accesses the whole 'super' field, while previous fields access parts of the super (eg super.mlx5_common)

Reply:
this field accesses only certain field under 'super' fields, those defined in  uct_rc_mlx5_common_config_table. Previous field access those defined in uct_ib_mlx5_iface_config_table

Reply:
need some cleanup here, so one field only will access 'super'. maybe unite/change inheritance of some structures